server/src/controllers/christmas-tree.es6

```diff
@@ -128,22 +128,24 @@ const postPayGov = xmlData => {
  */
 const permitResult = permit => {
   return {
-    permitId: permit.permitId,
-    orgStructureCode: permit.orgStructureCode,
-    firstName: permit.firstName,
-    lastName: permit.lastName,
-    emailAddress: permit.emailAddress,
-    quantity: permit.quantity,
-    totalCost: permit.totalCost,
-    status: permit.status,
+    ..._.pick(permit, [
+      'permitId',
+      'orgStructureCode',
+      'firstName',
+      'lastName',
+      'emailAddress',
+      'quantity',
+      'totalCost',
+      'status',
+      'paygovTrackingId',
+    ]),
     transactionDate: permit.updatedAt,
```

Comment:
is there a reason why these wouldn't be in the .pick?

Reply:
`pick` just grabs that key/value pair but doesn't change the key or value. The remaining entries include a change in either key or value in the result. We could 'pick' all of them and then do the transformation after. Ideally I like the functional version of lodash for this sort of stuff because you can chain these operations.

FYI: I noticed that while Lodash is already included as a dependency by the server, WE are not using it anywhere else, looks like its a dependency of several packages we are using.

Reply:
ah i see now, that's helpful. Generally, i'd like to avoid lodash and do native es6 transformations, but this could be helpful. It's also interesting that you don't have to do object assign.

Reply:
fair enough. We can definitely be more explicit about some of this without using Lodash, just by separating out the logic. The other good use case which didn't end up being used in this example is `get` which alls one to safely access nested fields ie:

```
output.applicantInfoEveningPhoneAreaCode =
    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.areaCode
      ? input.applicantInfo.eveningPhone.areaCode
      : null;
```
becomes
```
output.applicantInfoEveningPhoneAreaCode = _.get('input.applicantInfo.eveningPhone.areaCode', null);
```

My other concerns involve where the transformations live (maybe they should be outside the controller) and some optimizations (minimizing calls to `moment`: https://github.com/18F/fs-open-forest-platform/blob/dev/server/src/controllers/noncommercial.es6#L196)

But I'm good closing this if there is nothing currently actionable.

Reply:
I definitely like that _.get! as we have nested objects everywhere. I'm not sure this warrants another ticket, but if we find a story that this makes sense, we probably want to include.